### PR TITLE
Set up unit testing

### PR DESCRIPTION
**Pull request by Astatium5**
Set up unit testing.
@TangyrenMidKing, please, test it on your machine and let me know if you have any questions or if something does not work.

### CODE REVIEW

**Comment by TangyrenMidKing** [comment]
all looks fine for me

File: tests/api/test_events.py [hunk 1]
```diff
@@ -12,7 +19,33 @@ class TestEventsBlueprint(BaseTestCase):
...
     def test_create_event(self):
-        pass
+        with mock.patch("firebase_admin.auth.verify_id_token") as magic_mock:
+            magic_mock.return_value = {
+                "uid": "6Fowt6pbYbJAU8IRUudDnntrYvfc",
+            }
+
+            res = self.client.post(
+                "/events/create_event",
+                headers={"Authorization": "token"},
```
Comment (TangyrenMidKing) [inline tests/api/test_events.py:29]: How are the token works? I saw that magic_mock.return_value contains a hardcoded uid, will that be the return value when you pass the "token" the Authorization? If yes, do we have to disable the @checkToken when we run the test?

**Review by TangyrenMidKing — APPROVED** [review]
All works on my side except the token stuff we discussed before. But I think we are good for now.

File: tests/api/test_events.py (hunk 1 above)
Comment (Astatium5) [reply tests/api/test_events.py:29]: We don't have to disable anything for unit testing. The token should be mocked with that `magic_mock.return_value`

**Pull request merged**